### PR TITLE
Feature/export file

### PR DESCRIPTION
**Pull request by Alejoboga20**
# Description

- Create Excel file from features.
- Add Epic field in features to be compatible with what stories on board expects.
- Add some UI improvements.

## Screenshots

image

image

image

### CODE REVIEW

File: src/utils/generateExcelFile.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  const dataToExport: StoryMapFile = features.map((feature) => ({
+    Theme: 'Product Definition',
+    Epic: feature.epic,
+    Title: feature.name,
+    Story: feature.description,
+    Status: 'ToDo',
+  }));
+
+  const workSheet = utils.json_to_sheet(dataToExport);
+  //const worksheet = XLSX.utils.json_to_sheet(dataToExport);
```
Comment (dbrosio3) [inline src/utils/generateExcelFile.ts:23]: I think this line can be removed

File: src/product/pages/HomePage.tsx [hunk 2]
```diff
@@ -172,19 +173,29 @@ const HomePage = () => {
...
+              />
+            ))}
+          </div>
+          <div className='flex gap-4'>
+            <Button variant='primary' disabled={isLoading} onClick={() => setIsAddingFeature(true)}>
+              Add custom feature
+            </Button>
+            <Button
+              variant='secondary'
+              onClick={() => generateExcelFile(features.features)}
```
Comment (dbrosio3) [inline src/product/pages/HomePage.tsx:193]: Could this operation throw an error? In that case, should we handle it?

**Review by dbrosio3 — APPROVED** [review]
Looks great!

File: src/product/pages/HomePage.tsx (hunk 2 above)
Comment (Alejoboga20) [reply src/product/pages/HomePage.tsx:193]: I think It couldn't fail as it doesn't involve external  code.

File: src/utils/generateExcelFile.ts (hunk 1 above)
Comment (Alejoboga20) [reply src/utils/generateExcelFile.ts:23]: yes, thanks for pointing that out

**Pull request merged**